README.md

```diff
@@ -86,8 +86,8 @@ You can configure an **opt-in** behaviour by specifying an operation whitelist l
 ```js
 // Inside the component handling the network events
 useApolloNetworkStatus({
-  shouldHandle: (action: NetworkStatusAction) =>
-    action.payload.operation.getContext().useApolloNetworkStatus === true
+  shouldHandleOperation: (operation: Operation) =>
+    operation.getContext().useApolloNetworkStatus === true
```

Comment:
Previously the user would theoretically be able to handle a `REQUEST` event, but not the corresponding `SUCCESS`/`ERROR` event. That would lead to an inconsistent state.

Allowing the user to only ignore / handle whole operations avoids this issue.

If a behaviour like this is really necessary, the user can still use the low-level `useApolloNetworkStatusReducer` to implement custom behaviour.